components.go

```diff
@@ -132,6 +132,10 @@ func (event *VEvent) SetLocation(s string, props ...PropertyParameter) {
 	event.SetProperty(ComponentPropertyLocation, ToText(s), props...)
 }
 
+func (event *VEvent) SetGeo(lat interface{}, lng interface{}, props ...PropertyParameter) {
+	event.SetProperty(ComponentPropertyGeo, ToText(fmt.Sprintf("%v;%v", lat, lng)), props...)
```

Comment:
@pjebs, it appears the `;` character can't be escaped in this string. It's causing problems, at least for Android SDK. Would it be better to just ensure lat and lng are numeric?

A related Android error:
```
com.google.android.calendar E/ICalProvider: Parsing iCal file
    net.fortuna.ical4j.data.ParserException: Error at line 18:For input string: "-37828883\"
```

The culprit line being in the format like:
```
GEO:-11111\;22222
```

Reply:
@alexclifford do it manually in your project. (You can revert my commit).
I'll fix up this PR.